### PR TITLE
Correction of args errors and CM empty

### PR DESCRIPTION
**Pull request by melhk**

### CODE REVIEW

File: cm/base/BaseCM/cm_output.py [hunk 1]
```diff
@@ -74,19 +74,17 @@ class CMOutput(Schema):
...
+    # data structure
+    return output_schema.load(data=output)
 
 
 def output_raster(raster_name, raster_fd):
     """Add a raster to the api."""
     files = {"file": (raster_name, raster_fd, "image/tiff")}
     try:
-        resp = requests.post(API_URL + "api/geofile/", files=files)
+        API_URL = os.environ.get("API_URL")
```
Comment (neodark) [inline cm/base/BaseCM/cm_output.py]: Why is this in the function and not a global data from the application ?

File: cm/example_empty/worker.py [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
 @app.task(base=cm_base.CMBase, bind=True, schema_path=schema_path)
-def new_cm(self, file: str, params: dict):
-    """New cm to test the integration work flow"""
-    status = process(file)
-    return status
+def cm_empty(self, selection: dict, rasters: list, params: dict):
+    ret = dict()
+    ret["graphs"] = {}
+    ret["geofiles"] = {}
+    ret["values"] = {}
```
Comment (neodark) [inline cm/example_empty/worker.py:14]: as it doesn't add extra information we can have a one liner here:
`ret = {"graphs":{}, "geofiles":{}, "values":{}}`

Comment (melhk) [reply cm/example_empty/worker.py:14]: I find it easier for the future "implementers" to understand what to do if it is already decomposed and not merged in one line.

File: cm/base/BaseCM/cm_output.py (hunk 1 above)
Comment (melhk) [reply cm/base/BaseCM/cm_output.py:86]: I don't know, I have only corrected a "local bug" for now.....

Comment (melhk) [reply cm/base/BaseCM/cm_output.py]: I don't know, I have only corrected a bug "locally" for now...

File: cm/example_empty/worker.py (hunk 2 above)
Comment (neodark) [reply cm/example_empty/worker.py:14]: Agreed due to the future coders level.

**Review by neodark — APPROVED** [review]
Looks good to me

**Pull request merged**